vividus-bdd-engine/src/main/java/org/vividus/bdd/steps/ComparisonRule.java

```diff
@@ -94,6 +94,11 @@ public static ComparisonRule parse(String sign)
                 .findFirst();
     }
 
+    public String toString()
+    {
+        return super.toString().replaceAll("_", " ").toLowerCase();
```

Comment:
since JDK 11 is still our baseline, `.replaceAll("_", " ")` -> `.replace('_', ' ')`